lib/index.js

```diff
@@ -143,7 +144,7 @@ function stringify(obj) {
   var ret = {};
   for (var prop in obj) {
     if (obj.hasOwnProperty(prop)) {
-      ret[prop] = obj[prop].toString();
+      if (obj[prop]) ret[prop] = obj[prop].toString();
```

Comment:
@tsholmes I can't add `reject(obj)` inside the `stringify` function because line 74. If I don't strip the `null` before we hit `stringify`, we will return `"{}"` in the event that there are only null properties :/ and I need to return an empty string if there is nothing to send from props.